src/DB/projects.json

```diff
@@ -2767,5 +2767,23 @@
       "tech": ["Python", "Beautiful Soup"]
     }
   ]
+},
+{
+  "github_username": "panditshivammishra",
+  "Social_media": {
+    "gitHub": "https://github.com/panditshivammishra",
+    "LinkedIn": "https://www.linkedin.com/in/shivam-mishra-42226522b/",
+    "Instagram": "",
+    "YouTube": "YOUR_YOUTUBE_ACCOUNT_LINK",
+    "Twitter": "YOUR_TWITTER_ACCOUNT_LINK"
+  },
+  "Projects": [
+    {
+      "link": "https://panditshivammishra.github.io/firstgame.github.io/",
```

Comment:
add a github repo link

also, make sure, to merge the pull request, it is essential that your projects include a contributing guide, code of conduct, license, and a readme. These requirements must be fulfilled without any exceptions.